Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml

```diff
@@ -873,6 +888,42 @@
     tags:
     - Fruit
 
+- type: entity
+  name: golden apple
+  parent: FoodProduceBase
+  id: FoodGoldenApple
+  description: It should be shaped like a cube, shouldn't it?
+  components:
+  - type: FlavorProfile
+    flavors:
+      - apple
+      - metallic
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 30
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 10
+        - ReagentId: Vitamin
+          Quantity: 4
+        - ReagentId: DoctorsDelight
```

Comment:
Just make this trico, DD is a mixed drink and this doesn't make any sense. Its also got no limit like trico does.

Reply:
Poppy have bicaridine that mixed chem. I don't think that it makes so much trouble. DD just fit more.